lib/handlers/pkg.put.js

```diff
@@ -144,6 +154,11 @@ class PkgPut {
             throw new HttpError(400, 'Bad request');
         }
 
+        const exist = await this._exist(org, name, version);
+        if (exist) {
+            throw new HttpError(409, 'Conflict');
```

Comment:
You should probably be able to use an http status code -> message lookup map for these. Isn’t there one built into one of the node core modules?